services/QuillLMS/app/queries/quill_big_query.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# All messages to BigQuery should use this module
+
+require 'google/api_client'
+require 'googleauth'
+require 'stringio'
+
+module QuillBigQuery
```

Comment:
Take or leave. Consider making this an `ApplicationService` with an `initialize` and `run` to follow a common pattern in our codebase.